src/router/router.ts

```diff
@@ -45,4 +45,23 @@ const router = createRouter({
     routes,
 })
 
+router.beforeEach((to, from, next) => {
+    let isConnected = Boolean(localStorage.getItem('userConnected'))
```

Comment:
Tu peux utiliser `!!` pour caster en boolean plutôt que `Boolean()` c'est moins verbeux

```suggestion
    let isConnected =!!localStorage.getItem('userConnected')
```